.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,148 @@
+---
+name: 🐛 Bug Report
```

Comment:
```suggestion
name: 🐛  Bug Report
```

Reply:
@kratsg Why do you want two spaces for all of these?

Reply:
Ah okay, that must be some bug with mac OS then as they render fine elsewhere. 

![spaces](https://user-images.githubusercontent.com/5142394/131867261-02e1542e-2bcb-4ec9-8fa2-d972e657060e.png)

Reply:
Does this also happen with Firefox?